transformerlab/routers/data.py

```diff
@@ -56,15 +56,17 @@ async def dataset_info(dataset_id: str):
 
     if d is None:
         return {}
-
     r = {}
     # This means it is a custom dataset the user uploaded
     if d["location"] == "local":
         dataset = load_dataset(path=dirs.dataset_dir_by_id(dataset_id))
         # print(dataset['train'].features)
         r["features"] = dataset["train"].features
     else:
-        ds_builder = load_dataset_builder(dataset_id)
+        if (dataset_id == "wikipedia"):
```

Comment:
I don't think we should hardcode an if statement for wikipedia in the code.  Instead let's add configuration to the data definition in the gallery and then add code to handle passing a configuration around.